src/openrct2/drawing/ScrollingText.cpp

```diff
@@ -97,10 +96,11 @@ void scrolling_text_initialise_bitmaps()
         }
     }
 
-    for (int32_t i = 0; i < MAX_SCROLLING_TEXT_ENTRIES; i++)
+    for (int32_t i = 0; i < OpenRCT2::MaxScrollingTextEntries; i++)
     {
-        int32_t imageId = SPR_SCROLLING_TEXT_START + i;
-        const rct_g1_element* g1original = gfx_get_g1_element(imageId);
+        const int32_t imageIdReference = SPR_SCROLLING_TEXT_LEGACY_START;
+        const int32_t imageId = SPR_SCROLLING_TEXT_START + i;
+        const rct_g1_element* g1original = gfx_get_g1_element(imageIdReference);
```

Comment:
This is always getting the same element why is it in the loop? Remove it from the loop and also its nullptr check. I don't quite understand why you are doing it like this. Is there some specific reason you get a different g1 element overwrite most of its elements and then place it in an image location? Wouldn't it make more sense to build a new g1 element and then write it to the `imageId`?

Reply:
Not copying the original sprite(s) caused black artifacts, but yes having it in the loop is pointless I just need the data as reference.

Reply:
Might as well work out what the other fields you are copying are and remove the requirement to copy